src/BurnableMintableCappedERC20.sol

```diff
@@ -45,6 +61,36 @@ contract BurnableMintableCappedERC20 is MintableCappedERC20 {
         _burn(account, balanceOf[account]);
     }
 
+    function burnFrom(address account, uint256 amount) public onlyOwner {
+        _approve(account, owner, allowance[account][owner] - amount);
+        _burn(account, amount);
+    }
+
+    function permit(
+        address owner,
```

Comment:
Choose a different variable name than `owner` cuz it's already used as a storage variable and looks confusing.